backend/map-resources/include/map-resources/resource-conversion-inl.h

```diff
@@ -402,6 +417,225 @@ bool convertDepthMapToPointCloud(
   return true;
 }
 
+template <typename PointCloudType>
+bool convertPointCloudToDepthMap(
+    const PointCloudType& point_cloud_C, const aslam::Camera& camera,
+    const bool use_openni_format, const bool create_range_image,
+    cv::Mat* depth_map, cv::Mat* image) {
+  CHECK(!(
+      camera.getType() == aslam::Camera::Type::kLidar3D && !create_range_image))
```

Comment:
Can you move the negation inside to make it a bit more readable. I.e. `camera.getType() != aslam::Camera::Type::kLidar3D || create_range_image` is more clear to me.